dev/Deployment/DeploymentManager.cpp

```diff
@@ -356,8 +356,8 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         wil::handle_wait(processInfo.hProcess);
 
         DWORD processExitCode{};
-        THROW_IF_WIN32_BOOL_FALSE_MSG(GetExitCodeProcess(processInfo.hProcess, &processExitCode), "CmdLine: %ls, processExitCode: %lu", cmdLine.get(), processExitCode);
-        RETURN_IF_FAILED_MSG(processExitCode, "DeploymentAgent exitcode:0x%X", processExitCode);
+        THROW_IF_WIN32_BOOL_FALSE_MSG(GetExitCodeProcess(processInfo.hProcess, &processExitCode), "CmdLine: %ls, processExitCode: %u", cmdLine.get(), processExitCode);
```

Comment:
This checks the boolean result of the call but then provides the process exit code that couldn't be retrieved on failure. On failure, it would be more helpful to have a message that provides the result of GetLastError.

Reply:
Line 359 reports if there's an error retrieving the exit code. If so the ...IF_WIN32_BOOL_FALSE... part tells WIL to report the GetLastError()

If we can _get_ the exit code line 360 notes if the child process' exit code is not-success
Line 361 all is well so we return S_OK